ontopy/ontology.py

```diff
@@ -343,12 +344,13 @@ def get_by_label(
                 label_annotations=label_annotations,
                 prefix=prefix,
             )
-            if len(entitylist) > 0:
-                return entitylist[0]
+            if len(entitylist) == 1:
+                return next(iter(entitylist))
```

Comment:
`entitylist` er ikke lenger en liste, men et sett. Burde vi endre navn på den?

Vi ønsker å sjekke om det finnes entitylist, ikke om den kun har et element:

```suggestion
            if entitylist:
                return entitylist.pop()
```